gg.py

```diff
@@ -1,12 +0,0 @@
-import os
```

Comment:
You've been exposed. A real programmer would never automate his tasks using some other language.

Reply:
:(